client/landing/stepper/utils/flow-redirect-handler.ts

```diff
@@ -1,17 +1,11 @@
-import {
-	BLOG_FLOW,
-	FREE_FLOW,
-	LINK_IN_BIO_FLOW,
-	VIDEOPRESS_FLOW,
-	SENSEI_FLOW,
-} from '@automattic/onboarding';
+import { BLOG_FLOW, FREE_FLOW, VIDEOPRESS_FLOW, SENSEI_FLOW } from '@automattic/onboarding';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 
 // Flows to redirect
 const REMOVED_TAILORED_FLOWS = [
 	{ flow: BLOG_FLOW, to: '/start:lang?' },
 	{ flow: FREE_FLOW, to: '/start/free:lang?' },
-	{ flow: LINK_IN_BIO_FLOW, to: '/start:lang?' },
+	{ flow: 'link-in-bio', to: '/start:lang?' },
```

Comment:
Shouldn't we keep using the `LINK_IN_BIO_FLOW` constant? Not probable, but it might change its value.

Reply:
We don't have this constant anymore since it isn't a flow anymore.